AI/AIInterface.cpp

```diff
@@ -262,6 +262,11 @@ namespace AIInterface {
             return 0;
         }
 
+        if (!empire->EnqueuableItem(BT_BUILDING, item_name, location_id)) {
+            ErrorLogger() << "IssueEnqueueBuildingProductionOrder : specified item_name and location_id that don't indicate an item that can be enqueued at that location";
```

Comment:
Would perhaps be more useful to also output the relevant item name and location in the log message.